met/docs/Users_Guide/reformat_point.rst

```diff
@@ -837,7 +836,145 @@ We will not give a detailed description of each CALIPSO data product that lidar2
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. The **ioda_file** argument is an input IODA NetCDF point observation file to be processed.
+
+2. The **netcdf_file** argument is the NetCDF output file to be written.
+
+Optional arguments for ioda2nc
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+3. The **-config config_file** is a IODA2NCConfig file to filter the point observations and define time summaries.
+
+4. The **-obs_var var_list** setting is a comma-separated list of variables to be saved from input the input file (by defaults, saves "all").
+
+5. The **-iodafile ioda_file** option specifies additional input IODA observation files to be processed.
+
+6. The **-valid_beg time** and **-valid_end time** options in YYYYMMDD[_HH[MMSS]] format overrides the retention time window from the configuration file.
+
+7. The  **-nmsg n** indicates the number of IODA records to process.
+
+8. The **-log** file option directs output and errors to the specified log file. All messages will be written to that file as well as standard out and error. Thus, users can save the messages without having to redirect the output on the command line. The default behavior is no log file.
+
+9. The **-v level** option indicates the desired level of verbosity. The value of “level” will override the default setting of 2. Setting the verbosity to 0 will make the tool run with no log messages, while increasing the verbosity above 1 will increase the amount of logging.
+
+10. The **-compress level** option indicates the desired level of compression (deflate level) for NetCDF variables. The valid level is between 0 and 9. The value of “level” will override the default setting of 0 from the configuration file or the environment variable MET_NC_COMPRESS. Setting the compression level to 0 will make no compression for the NetCDF output. Lower number is for fast compression and higher number is for better compression.
+
+An example of the ioda2nc calling sequence is shown below:
+
+.. code-block:: none
+
+    ioda2nc \
+    ioda.NC001007.2020031012.nc ioda2nc.2020031012.nc \
+    -config IODA2NCConfig -v 3 -lg run_ioda2nc.log
       
+In this example, the IODA2NC tool will reformat the data in the input ioda.NC001007.2020031012.nc file and write the output to a file named ioda2nc.2020031012.nc. The data to be processed is specified by IODA2NCConfig, log messages will be written to the ioda2nc.log file, and the verbosity level is three.
+
+
+ioda2nc configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default configuration file for the IODA2NC tool named **IODA2NcConfig_default** can be found in the installed *share/met/config* directory. It is recommended that users make a copy of this file prior to modifying its contents.
+
+The IODA2NC configuration file is optional and only necessary when defining filtering the input observations or defining time summaries. The contents of the default IODA2NC configuration file are described below.
+
+__________________
+
+.. code-block:: none
```

Comment:
I see there is a line above the code blocks.  In the ".. code-block:: none" statements, could you please try changing those to be ".. code-block:: ini" and see if that gets rid of the line?  If not, it's fine and I'll look into what's going on.

Reply:
The line above this code block looks correct to me. We use that horizontal line between the config file entries being described. It matches the formatting used for the other point pre-processing tools, as well as config entries in Point-Stat (for example):
https://met.readthedocs.io/en/feature_1731_docs/Users_Guide/point-stat.html#point-stat-configuration-file

Reply:
Thanks for letting me know.  I did not realize that.